### DISCUSSION TITLE
Migration in progress...

### DISCUSSION THREAD

**Discussion by mirek-z** [hassio-addons/addon-unifi#331] [General]
Hello
I have a problem. an attempt to start UN ends with the message: Database Migration in progress ... and so on.
anyone have an idea how to fix it?

**Comment by MyRandomUsername** [comment]
This seems to happen to people who had a 32 bit OS and upgraded to 64 bit. I had to downgrade to 0.23 addon version, get a backup of my config. Then completely uninstall, install fresh and import that config back in.
